src/jupyter_contrib_nbextensions/nbextensions/hide_input_all/readme.md

```diff
@@ -18,3 +18,6 @@ Internals
 
 The codecell hiding state is stored in the metadata `IPython.notebook.metadata.hide_input`.
 If it is set to `true`, all codecells will be hidden on reload.
+
+Exporting using `nbconvert` is currently not supported.
```

Comment:
I think the same metadata value is set as in hide_input, in which case the same export advice as there should work? Or perhaps I've misunderstood/misremembered